applications/serial_lte_modem/src/slm_util.h

```diff
@@ -55,6 +55,16 @@ bool slm_util_cmd_casecmp(const char *cmd, const char *slm_cmd);
  */
 bool slm_util_hex_check(const uint8_t *hex, uint16_t hex_len);
 
+/**
+ * @brief Detect hexdecimal string data type
+ *
+ * @param[in] string arrary to be checked
```

Comment:
The param name does not match the declaration
```suggestion
 * @param[in] hex array to be checked
```

Reply:
I've modified the param name. Many thanks!